.tekton/windows-machine-config-operator-bundle-master-pull-request.yaml

```diff
@@ -21,8 +21,6 @@ spec:
   params:
   - name: hermetic
     value: "true"
-  - name: build-source-image
-    value: "true"
```

Comment:
I'm not sure what the pros and cons are, but you could perhaps just set this to "false" and leave the task in place.

Reply:
feels weird leaving 'dead code', but your suggestion seems to be the preferred way

Reply:
@sebsoto  the default value is `false` already, consider removing the param at all

https://github.com/openshift/windows-machine-config-operator/pull/2800/files#diff-b49cb239cae2ed6898c3b15047aca512487e62542d6e7ba06b1605ba1145008dR104